flocker/node/test/test_deploy.py

```diff
@@ -493,20 +493,19 @@ def test_link_environment(self):
         ``<alias>_PORT_<local_port>_<protocol>`` and the broken out variants
         ``_ADDR``, ``_PORT`` and ``_PROTO``.
         """
-
         environment = _link_environment(
             protocol="udp",
-            alias="dash-alias",
+            alias="dashalias",
             local_port=80,
             hostname=u"the-host",
             remote_port=8080)
         self.assertEqual(
             environment,
             {
-                u'DASH_ALIAS_PORT_80_UDP': u'udp://the-host:8080',
-                u'DASH_ALIAS_PORT_80_UDP_PROTO': u'udp',
-                u'DASH_ALIAS_PORT_80_UDP_ADDR': u'the-host',
-                u'DASH_ALIAS_PORT_80_UDP_PORT': u'8080',
+                u'DASHALIAS_PORT_80_UDP': u'udp://the-host:8080',
+                u'DASHALIAS_PORT_80_UDP_PROTO': u'udp',
+                u'DASHALIAS_PORT_80_UDP_ADDR': u'the-host',
+                u'DASHALIAS_PORT_80_UDP_PORT': u'8080',
             })
```

Comment:
I guess `_link_environment` doesn't care about the protocol you tell it to use...  It would probably make more sense to exercise TCP here since that's what we want to work... But I don't think it actually matters.